src/PhpSpec/Listener/NamedConstructorNotFoundListener.php

```diff
@@ -79,6 +79,17 @@ public function afterSuite(SuiteEvent $event)
                     'arguments' => $arguments
                 ));
                 $event->markAsWorthRerunning();
+
+                if (!method_exists($classname, '__construct')) {
```

Comment:
At the moment if the author has put an existing public constructor in, it might not be appropriate to modify it to private so I'm completely skipping if any constructor exists.